src/lld/backend_patterns/repository/template/Correction.md

```diff
@@ -4,6 +4,76 @@
 # DTO question
 - Q: What is the best practises for DTO objects for 2 or more entity relation?
 - A: DTO can have builder pattern to build required inputs if it seems to more combinations
```

Comment:
Can it be added with a example or link for quick recap

Reply:
https://docs.google.com/document/d/1_Ql9inqZZX-7tJAkUqnrI6lpdXUmFr3VW0QtARhrSgM/edit